cluster-autoscaler/utils/drain/drain.go

```diff
@@ -29,8 +29,8 @@ import (
 )
 
 const (
-	// PodDeletionTimeout - maximum time after which a to be deleted pod is not included in the list of pods for drain.
-	PodDeletionTimeout = 5 * time.Minute
+	// PodDeletionTimeout - time after which a pod to be deleted is not included in the list of pods for drain.
+	PodDeletionTimeout = 10 * time.Minute
```

Comment:
10 is graceful termination. Please allow a bit more time  - 12 min or so.

Reply:
Done